common.tf

```diff
@@ -4,7 +4,7 @@ data "aws_vpc" "apiary_vpc" {
 
 data "aws_route53_zone" "ranger_zone" {
   name   = "${var.ranger_domain_name}"
-  vpc_id = "${var.vpc_id}"
+  private_zone = true
```

Comment:
Both the old line 7 and the new line 7 here specify that the zone is a private zone.  However, if you are running the Terraform from a different region than the zone is associated with, the old line won't work, because the VPC id of the other region won't be associated with the zone.  This change makes it work cross-zone.